csharp/ql/test/library-tests/standalone/externalLocationSink/standalone.cs

```diff
@@ -0,0 +1,27 @@
+using A;
+using B;
+
+namespace A
+{
+    public interface ILogger { }
+}
+
+namespace B
+{
+    public interface ILogger { }
+}
+
+public class C
+{
+    public ILogger logger;
```

Comment:
OOI, which type does Roslyn then assign to this field?

Reply:
In this case it's `A.ILogger`. What's odd is that when we process the field, there's no sign of ambiguity.

Reply:
That's also what I don't understand, I would have expected that Roslyn didn't assign any type at all. Or is it perhaps because we just take the first candidate symbol?

Reply:
No, it's not us.

Reply:
To conclude, we found that there's no ambiguity when processing the field or the reference to the field. But the field's type symbol is actually an ambiguous error symbol.